src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -613,6 +618,7 @@ class OverflowMenu extends Component {
     );
 
     const overflowMenuOptionsClasses = classNames(
+      this.props.menuOptionsClass,
```

Comment:
Unless you put `menuOptionClass` to [the code destructing `props`](https://github.com/IBM/carbon-components-react/blob/v7.0.0/src/components/OverflowMenu/OverflowMenu.js#L562-L581), the props goes to [the trigger `<div>`](https://github.com/IBM/carbon-components-react/blob/v7.0.0/src/components/OverflowMenu/OverflowMenu.js#L696), too, which we don't want - That said, could you use the variable destructed from `prop` here instead? Thanks!

Reply:
good spot! have made the suggested changes 👍